ui/components/ProgressBar/ProgressBarStyle.js

```diff
@@ -0,0 +1,82 @@
+import styled from "styled-components";
+
+const ProgressBarStyle = styled.div`
```

Comment:
lets create different styled components for each component such as `li`, `ul` , `timeline` , `status` etc. rather than putting all of them in the same div

Reply:
@asubedy , this is an example of how styled components can be implemented.

Reply:
Ok so `make styles` and `styled-components` are different? This way can we style each components?

Reply:
Yes.